components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityMgtEventListener.java

```diff
@@ -1732,9 +1735,11 @@ private boolean isUserExistsInDomain(UserStoreManager userStoreManager, String u
         return isExists;
     }
 
-    private boolean isAuthPolicyAccountExistCheck() {
+    private void setUserExistThreadLocal() {
 
-        return Boolean.parseBoolean(IdentityUtil.getProperty("AuthenticationPolicy.CheckAccountExist"));
+        IdentityUtil.threadLocalProperties.get().put(USER_EXIST_THREAD_LOCAL_PROPERTY, true);
+        if (log.isDebugEnabled()) {
+            log.debug(USER_EXIST_THREAD_LOCAL_PROPERTY + " is added as true to the thread local.");
```

Comment:
log the username and user store domain.

Reply:
Fixed https://github.com/wso2-extensions/identity-governance/pull/436/commits/246891b1c328142a14634b1782d6680737e570ac